src/utils/strategies/BaseStrategy.js

```diff
@@ -23,8 +23,6 @@ export default class BaseStrategy {
       throw new Error('startY is out of range');
     } else if (endX > documentWidth) {
       throw new Error('endX is out of range');
-    } else if (endY > documentHeight) {
```

Comment:
If we are passing options, I would disabling this error should be in the option as well. By removing this, this is breaking expected behavior (though I think most people would be okay with it)